src/service/main.cpp

```diff
@@ -120,10 +121,11 @@ int main(int argc, char **argv) {
         grc = grcBuffer.str();
     }
 
+    Digitizer::Settings settings;
     Broker broker("/PrimaryBroker");
     // REST backend
     auto fs           = cmrc::assets::get_filesystem();
-    using RestBackend = FileServerRestBackend<PLAIN_HTTP, decltype(fs)>;
+    using RestBackend = FileServerRestBackend<HTTPS, decltype(fs)>;
```

Comment:
For the wasm ui the `PLAIN_HTTP` is not useful, but it might be interesting for dumb http to access the e.g. settings or acquisition endpoints via http. Is it a problem to serve http and https at the same time? I don't think that's urgently needed, but would be good to know if it is easily doable.

Reply:
Technically, we should be able to serve http and https on the same port, but  our RestBackend is not prepared for  that and that would be confusing.  We could open another port and serve unencrypted content there. Of course, that requires absolute urls.

Reply:
No, not on the same port. This would not be for usage by the UI but rather if I just have a small script that wants to fetch e.g. some setting. Not critical for this PR, just want to know whether this would mean changing the RestBackend or if we could also just spawn 2 rest backends.